lib/org/chromium/webidl/Diff.js

```diff
@@ -276,19 +276,13 @@ foam.CLASS({
           var leftProps = left.cls_.getAxiomsByClass(foam.core.Property);
           var rightProps = right.cls_.getAxiomsByClass(foam.core.Property);
 
-          // Determine if left and right have the same properties.
-          // FUTURE: Relax condition and iterate over sorted collections of
-          // left and right properties.
-          if (foam.util.compare(leftProps, rightProps) !== 0) {
-            // Cannot perform a deep comparison of the objects.
-            this.addChunk_(id, this.DiffStatus.VALUES_DIFFER, left,
-                right, changes);
-            return;
-          }
+          // Perform a diff on all common properties.
```

Comment:
At this point of the code, it has been determined that they are both FOAM Objects, however, these objects could be instances of different classes. These should be instances of the same class most of the time, but there are cases where they are different (e.g. `Literal` and `Identifier` have been found for the same properties).